packages/flutter_tools/lib/src/asset.dart

```diff
@@ -206,22 +209,23 @@ class ManifestAssetBundle implements AssetBundle {
   Future<int> build({
     String manifestPath = defaultManifestPath,
     String? assetDirPath,
+    FlutterProject? flutterProject,
```

Comment:
you'll need to make a g3fix for this change, since google3 has a class that extends this one

Reply:
This is a new parameter for `ManifestAssetBundle`, not `AssetBundle` (which the google3 one extends), so I think no update is needed.